pkgs/tools/networking/vpnc-scripts/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchgit
+, makeWrapper
+, nettools, gawk, openresolv, coreutils, gnugrep
```

Comment:
Great work so far👍. Unfortunately the new dependency of [`openresolv`](https://github.com/NixOS/nixpkgs/blob/e7e72e070b9a82730e97134bf33d6a7e608a6bf3/pkgs/tools/networking/openresolv/default.nix#L38) breaks the build on macOS which is unfortunate as it was working before.

Can/does `openresolv` also work on macOS?

Reply:
Oh, ugh.  I don't have a mac so didn't realize that -- sorry!

I don't see any mention of darwin/macOS on the openresolv homepage.  I think it's likely that it's not necessary, though.  The design of `vpnc-script` seems to be to test at runtime for the availability of various ways of modifying `/etc/resolv.conf`, with `openresolv` being only one of those.  I would bet that one of the other approaches will work on macOS, so it wouldn't matter if `openresolv` were unavailable when building on darwin.

The trick is how to make that happen.  I've never tried to make a nix expression's dependencies vary by platform.  Do you have any pointers on how that might be done?